install/installer/pkg/config/v1/validation.go

```diff
@@ -125,6 +125,7 @@ func (v version) ClusterValidation(rcfg interface{}) cluster.ValidationChecks {
 	var res cluster.ValidationChecks
 	res = append(res, cluster.CheckSecret(cfg.Certificate.Name, cluster.CheckSecretRequiredData("tls.crt", "tls.key")))
 
+	res = append(res, cluster.CheckSecret(cfg.PersonalAccessTokenSigningKey.Name, cluster.CheckSecretRequiredData("personal-access-token-signing-key")))
```

Comment:
@easyCZ Added validation for both the secret and key (Looking at the format in the[ preview environments file](https://github.com/gitpod-io/gitpod/blob/tar/installer-mandate-15028/.werft/jobs/build/public-api/personal-access-token-signing-key.yaml#L3)). Do you see a reason to rename the key? 🤔

Reply:
Not really. I think we can stick with this name for now.